### PR TITLE
Adapt risk rating type to the oxo one.

### PR DESCRIPTION
**Pull request by deadly-panda**

### CODE REVIEW

File: src/ostorlab/serve_app/types.py [hunk 1]
```diff
@@ -120,6 +128,15 @@ def resolve_detail(
                 ],
             )
 
+    def resolve_risk_rating(
```
Comment (PiranhaSa) [inline src/ostorlab/serve_app/types.py:131]: Does this need unit test ?

**Review by PiranhaSa — APPROVED** [review]

File: src/ostorlab/serve_app/types.py [hunk 2]
```diff
@@ -59,11 +60,18 @@ class OxoKnowledgeBaseVulnerabilityType(graphene.ObjectType):
...
+    [(risk.name.upper(), i) for i, risk in enumerate(utils_rik_rating.RiskRating)],
+)
+
+
 class OxoVulnerabilityType(graphene_sqlalchemy.SQLAlchemyObjectType):
     """SQLAlchemy object type for a vulnerability."""
 
     detail = graphene.Field(OxoKnowledgeBaseVulnerabilityType, required=False)
     cvss_v3_base_score = graphene.Float(required=False)
+    risk_rating = graphene.Field(OxoRiskRatingEnum, required=False)
```
Comment (benyissa) [inline src/ostorlab/serve_app/types.py:74]:
```suggestion
    risk_rating = graphene.Field(OxoRiskRatingEnum)
```
It's False by default

File: src/ostorlab/serve_app/types.py [hunk 3]
```diff
@@ -120,6 +128,15 @@ def resolve_detail(
...
             )
 
+    def resolve_risk_rating(
+        self: models.Vulnerability, info: graphql_base.ResolveInfo
+    ) -> OxoRiskRatingEnum:
+        """Resolve risk rating of vulnerability"""
+        try:
+            return OxoRiskRatingEnum[self.risk_rating.name]
+        except KeyError:
+            return None
```
Comment (mohsinenar) [inline src/ostorlab/serve_app/types.py:138]: Update the return type please! 

OxoRiskRatingEnum | None

**Review by mohsinenar — CHANGES_REQUESTED** [review]

File: src/ostorlab/serve_app/types.py (hunk 1 above)
Comment (deadly-panda) [reply src/ostorlab/serve_app/types.py:131]: There is already one.. (independent from name of field type)

File: src/ostorlab/serve_app/types.py (hunk 2 above)
Comment (oussamaessaji) [reply src/ostorlab/serve_app/types.py:74]: explicit!!

**Review by 3asm — APPROVED** [review]

File: src/ostorlab/serve_app/types.py (hunk 3 above)
Comment (3asm) [reply src/ostorlab/serve_app/types.py]: +1

**Review by 3asm — APPROVED** [review]

**Pull request merged**